frontend/templates/project/project_index.html

```diff
@@ -4,38 +4,38 @@
     <title>Projects | Research Software Directory</title>
 {%- endblock -%}
 {%- block content -%}
-
-<section class="" id="title">
-    <div class="container">
-        <div class="row">
-	    <div class="text">
-		<h1 class="title">Projects</h1>
-		<p>This list of projects of the Netherlands eScience Center contains {{titles|length}} projects</p>
-            </div>
-	</div>
-    </div>
-</section>
-
-<section class="" id="projects">
-    <div class="container">
-        <div class="row">
-	    <div class="col-1-1">
-                <div class="row project-items">
-                    {% for title in (titles  | sort(attribute="title")) %}
-		        <article class="project-item col-1-2"><a href="/projects/{{ title.id }}">
-			    <div class="content">
-				<div class="thumb"><img src="{{title.imageUrl}}" alt=""></div>
-				    <div class="text">
-				        <p class="text-blue">{{ title.title }} ({{ title.yearStart }}{% if title.yearStart != title.yearEnd %}-{{ title.yearEnd }}{% endif %})</p>
-				        <p>{{title.subtitle}}</p>
-				    </div>
-			    </div>
-		        </a></article>
-                    {% endfor %}
+<div id="project_page">
+	<section class="" id="title">
+		<div class="container">
+			<div class="row">
```

Comment:
Can you correct the indentation, it looks a bit sloppy now

Reply:
done

Reply:
thanks